src/com/javabasic/lesson6/homework/Task2.java

```diff
@@ -0,0 +1,29 @@
+package com.javabasic.lesson6.homework;
+
+import java.util.Scanner;
+
+public class Task2 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        double numberOne = Double.parseDouble(scanner.nextLine());
+        String action = scanner.nextLine();
+        double numberTwo = Double.parseDouble(scanner.nextLine());
+
+        System.out.println(getResult(numberOne, action, numberTwo));
+    }
+
+    public static double getResult(double numberOne, String  action, double numberTwo) {
+        double result = switch (action) {
+            case "+" -> numberOne+numberTwo;
+            case "-" -> numberOne-numberTwo;
+            case "/" -> numberOne/numberTwo;
```

Comment:
I suggest validating numberTwo.
What if numberTwo is zero and you choose / operation?